OWML.ModHelper.Interaction/ModInteraction.cs

```diff
@@ -0,0 +1,82 @@
+﻿using OWML.Common;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace OWML.ModHelper.Interaction
+{
+    public class ModInteraction : IModInteraction
+    {
+        private readonly IList<IModBehaviour> _modList;
+        private Dictionary<string, List<IModBehaviour>> _dependantDict = new Dictionary<string, List<IModBehaviour>>();
+        private Dictionary<string, List<IModBehaviour>> _dependencyDict = new Dictionary<string, List<IModBehaviour>>();
+
+        public ModInteraction(IList<IModBehaviour> list)
+        {
+            _modList = list;
+
+            RegenDicts();
+        }
+
+        private void RegenDicts()
+        {
+            _dependantDict = new Dictionary<string, List<IModBehaviour>>();
+            _dependencyDict = new Dictionary<string, List<IModBehaviour>>();
+            foreach (var mod in _modList)
+            {
+                var dependants = new List<IModBehaviour>();
+                var dependencies = new List<IModBehaviour>();
+                foreach (var dependency in _modList)
+                { 
+                    if (dependency.ModHelper.Manifest.Dependencies.Any() && dependency.ModHelper.Manifest.Dependencies.Contains(mod.ModHelper.Manifest.UniqueName))
```

Comment:
Pretty sure you can remove `dependency.ModHelper.Manifest.Dependencies.Any() && ` and it'll work the same.